XamarinCommunityToolkitSample/Xamarin.CommunityToolkit.Sample.csproj

```diff
@@ -10,6 +10,15 @@
     <Compile Remove="resources\**" />
     <EmbeddedResource Remove="resources\**" />
     <None Remove="resources\**" />
+    <EmbeddedResource Update="Resx\AppResources.es.resx">
```

Comment:
I didn't actually pull down this code yet, but are the resource files under the main resource file? Else we will have a lot of resource files under there, there is that way where you can add them as children to the main resource file. You know what I mean? 😬

Reply:
The files are inside to Resx folder https://puu.sh/GlibY/319f8dfbc7.png

Reply:
Not entirely what I meant, but doesn't matter. We'll figure it out later, or not ;)

Reply:
🏃🏻‍♀️🏃🏻‍♀️ 😜